### PR TITLE
CLDR-14647 Modern Dashboard Widget

### PR DESCRIPTION
**Pull request by btangmu**
-Display Dashboard notification types like Error as circled E

-Include comments for Dashboard items (errors/warnings/...)

-Add a button for opening Dashboard; hide button when Dash open

-Change gear menu to three-line menu

##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-14647
- [x] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

**Comment by btangmu** [comment]
I just pushed one correction: `this.data` not just `data`

File: tools/cldr-apps/js/src/esm/cldrTable.js [hunk 1]
```diff
@@ -392,7 +389,7 @@ function getSingleRowUrl(tr, theRow) {
   p.append("xpath", theRow.xpathId);
   p.append("fhash", tr.rowHash);
   p.append("automatic", "t");
-  if (cldrStatus.isDashboard()) {
+  if (true || cldrStatus.isDashboard()) {
```
Comment (srl295) [inline tools/cldr-apps/js/src/esm/cldrTable.js:392]: leftover?

File: tools/cldr-apps/js/src/esm/cldrText.js [hunk 2]
```diff
@@ -437,14 +437,14 @@ const strings = {
...
   special_forum_participation: "Forum Participation",
-  special_gear: "⚙",
   special_general:
     "Please hover over the sidebar to choose a section to begin entering data. If you have not already done so, please read the <a target='_blank' href='http://www.unicode.org/cldr/survey_tool.html'>Instructions</a>, particularly the Guide and the Walkthrough. You can also use the Dashboard to see all the errors, warnings, and missing items in one place.",
   special_list_emails: "List Email Addresses",
   special_list_users: "List Users",
   special_locales: "Locale List",
   special_lookup: "Look up a code or xpath",
   special_mail: "Notifications (SMOKETEST ONLY)",
+  special_menu: "☰",
```
Comment (srl295) [inline tools/cldr-apps/js/src/esm/cldrText.js:447]: 🍔 !
Is a trigram the best way to do this? would it be better as `glyphicons-menu`?

File: tools/cldr-apps/js/src/views/DashboardWidget.vue [hunk 3]
```diff
@@ -26,7 +26,7 @@
           title="Close"
           @click="closeDashboard"
         >
-          x
+          X
```
Comment (srl295) [inline tools/cldr-apps/js/src/views/DashboardWidget.vue:29]: better as `glyphicon-remove-sign` ?

**Review by srl295 — APPROVED** [review]
lgtm with questions, could be followups

File: tools/cldr-apps/js/src/esm/cldrTable.js (hunk 1 above)
Comment (btangmu) [reply tools/cldr-apps/js/src/esm/cldrTable.js:392]: Actually that's the beginning of further work -- updating the dash after the user votes. Currently isDashboard is always false for the dash widget but we need to make the request as though it's true

File: tools/cldr-apps/js/src/esm/cldrText.js (hunk 2 above)
Comment (btangmu) [reply tools/cldr-apps/js/src/esm/cldrText.js:447]: maybe, though glyphicons depends on bootstrap which is currently ancient, so reluctant to add more bootstrap dependencies now

File: tools/cldr-apps/js/src/views/DashboardWidget.vue (hunk 3 above)
Comment (btangmu) [reply tools/cldr-apps/js/src/views/DashboardWidget.vue:29]: maybe -- let's discuss, as it relates to bootstrap, fonts...

**Pull request merged**